build-system/eslint-rules/camelcase.js

```diff
@@ -46,6 +46,14 @@ module.exports = function (context) {
         }
       }
 
+      // Permit object destructuring, since that is similar to membership access.
```

Comment:
Shouldn't we only permit destructer-rename, that way we avoid allowing standalone identifiers to take on the bad format?

Reply:
Yup, good catch.